dash/dash.py

```diff
@@ -170,6 +170,9 @@ def __init__(
         self._meta_tags = meta_tags or []
         self._favicon = None
 
+        # default renderer string
+        self.renderer = 'var renderer = new DashRenderer();'
```

Comment:
FYI @valentijnnieman - the hazards of a 6-month-old PR, this default was being set again and again in `_add_url`... no reason for that, right?

Reply:
Oh weird - no I doubt that that would be useful! Sorry about that.